docs/kyma-environment-broker/03-03-runtime-operations.md

```diff
@@ -7,7 +7,23 @@ Kyma Environment Broker allows you to configure operations that you can run on a
 
 > **NOTE:** It's important to set lower timeouts for the Kyma installation in the Runtime Provisioner.
 
-## Provisioning
+## Operations
+
```

Comment:
I think we need to describe what stage it is:
 - once stage is finished, won't be repeated (in opposite to step)
 - stage groups steps
do not describe here operation nor steps (it is described at the end of the doc)